### PR TITLE
Adds Chilling effects to Slime Crossbreeding

### PR DESCRIPTION
**Pull request by AmatsuDF**
This PR adds in the Chilling effects from Slime Crossbreeding, with associated icons and code files. All content is from upstream and not modulized, as I saw no need to do so with content directly from upstream.

Some compromises had to be made, though: as /tg/station has newer, and different defines and variables in places, I had no choice but to comment out a few lines here and there. If a proper solution is available, I am open to suggestions.

This is a port of [#40928](https://github.com/tgstation/tgstation/pull/40928) from /tg/station, slightly modified to account for Citadel specific differences.

### CODE REVIEW

File: code/modules/research/xenobiology/crossbreeding/_weapons.dm [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	w_class = WEIGHT_CLASS_HUGE
+	force = 5
+	max_charges = 1 //Recharging costs blood.
+	recharge_rate = 1
+	ammo_type = /obj/item/ammo_casing/magic/bloodchill
+	fire_sound = 'sound/effects/attackblob.ogg'
+
+/obj/item/gun/magic/bloodchill/Initialize()
+	. = ..()
+//	add_trait(TRAIT_NODROP, HAND_REPLACEMENT_TRAIT) //Commented out because Citadel lacks add_trait, TRAIT_NODROP and HAND_REPLACEMENT_TRAIT.
```
Comment (deathride58) [inline code/modules/research/xenobiology/crossbreeding/_weapons.dm]: Replace with `ITEM_FLAGS = NODROP` as a var, then remove this initialize() proc override

**Review by deathride58 — DISMISSED** [review]

File: code/modules/research/xenobiology/crossbreeding/_weapons.dm (hunk 1 above)
Comment (AmatsuDF) [reply code/modules/research/xenobiology/crossbreeding/_weapons.dm]: Done.

**Comment by Trilbyspaceclone** [comment]
/obj/item/clothing/suit/armor/heavy
	name = "heavy armor"
	desc = "A heavily armored suit that protects against moderate damage."
	icon_state = "heavy"
	item_state = "swat_suit"
	w_class = WEIGHT_CLASS_BULKY
	gas_transfer_coefficient = 0.9
	clothing_flags = THICKMATERIAL
	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
	slowdown = 3
	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
	armor = list("melee" = 80, "bullet" = 80, "laser" = 50, "energy" = 50, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 90, "acid" = 90)

Ahahah

File: code/modules/research/xenobiology/crossbreeding/_clothing.dm [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+/obj/item/clothing/suit/armor/heavy/adamantine
+	name = "adamantine armor"
+	desc = "A full suit of adamantine plate armor. Impressively resistant to damage, but weighs about as much as you do."
+	icon_state = "adamsuit"
+	item_state = "adamsuit"
+	flags_inv = NONE
+	obj_flags = IMMUTABLE_SLOW
+	slowdown = 4
+	var/hit_reflect_chance = 40
+
```
Comment (Trilbyspaceclone) [inline code/modules/research/xenobiology/crossbreeding/_clothing.dm:139]: Make its armor not that of DS gear

Comment (AmatsuDF) [reply code/modules/research/xenobiology/crossbreeding/_clothing.dm:139]: I think the balancing factor is that while extremely durable, it also makes you about as slow as a snail and provides no head protection. It's also not innately space proof.

Comment (AmatsuDF) [reply code/modules/research/xenobiology/crossbreeding/_clothing.dm:139]: Changed.

**Review by silicons — APPROVED** [review]

**Pull request merged**